doc/source/guidelines/logging.rst

```diff
@@ -252,11 +250,11 @@ The instance loggers can be accessed in two places:
 * ``LOG._instances``. This field is a ``dict`` where the key is the
   name of the created logger.
 
-These instance loggers inherit from the ``pymapdl_global`` output
-handlers and logging level unless otherwise specified.  The way this
-logger works is very similar to the global logger.  You can add a file
-handler if you wish using the method ``log_to_file`` or change the log
-level using :meth:`logging.Logger.setLevel`.
+These instance loggers inherit from the ``pymapdl_global`` output handlers and
+logging level unless otherwise specified.  The way this logger works is very
+similar to the global logger.  You can add a file handler if you wish using the
+method ``log_to_file`` or change the log level using
+:meth:`logging.Logger.setLevel`.
```

Comment:
```suggestion
These instance loggers inherit from the ``pymapdl_global`` output handlers and
logging level unless otherwise specified. The way this logger works is very
similar to the global logger. You can add a file handler if you wish using the
method ``log_to_file`` or change the log level using
:meth:`logging.Logger.setLevel`.
```